JordanPhillips_ImJKP/hw_2/array_exercises.rb

```diff
@@ -0,0 +1,52 @@
+# Find the length of strings in an array
+
+def length_finder(input_array)
+  lengths=[]
+  input_array.each do |item| 
+    if item.is_a? String
+      lengths << item.length
+    end
+  end
+  lengths
+end
+
+# Find the frequency of a string in a sentence
+
+def find_frequency(sentence, word)
+  s = sentence.downcase.split(" ")
+  w = word.downcase
+  s.count(w)
+end
+
+# Select random elements from an array
+
+def random_select(array, n)
+  n.times do
+    puts array[rand(array.length-1)]
+  end
+end
```

Comment:
Check this fancy method out: http://ruby-doc.org/core-2.2.0/Array.html#method-i-sample